src/state/pnews/hooks.ts

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios'
+import qs from 'qs'
+import { useQuery } from 'react-query'
+
+export interface News {
+  id: number
+  title: string
+  content: string
+  createdAt: Date
+  updatedAt: Date | null
+}
+
+// Get News in Pangolin Strapi api
+export function useGetNews() {
+  const query = qs.stringify(
+    {
+      sort: ['-date_created'],
+      limit: 10
+    },
+    {
+      encodeValuesOnly: true
+    }
+  )
+
+  return useQuery('getNews', async () => {
+    const response = await axios.get(`https://p7gm7mqi.directus.app/items/news?${query}`, {
```

Comment:
lets move this url to some constant file

Reply:
done